SonarQube.Client/Api/V5_50/DownloadStaticFile.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SonarQube.Client.Requests;
+
+namespace SonarQube.Client.Api.V5_50
+{
+    public class DownloadStaticFile : RequestBase<Stream>, IDownloadStaticFile
+    {
+        [JsonIgnore]
+        public string PluginKey { get; set; }
+
+        [JsonIgnore]
+        public string FileName { get; set; }
+
+        protected override string Path => $"static/{PluginKey}/{FileName}";
```

Comment:
The arguments are part of the path, hence they are not serialized as query string parameters.